go.mod

```diff
@@ -24,15 +23,15 @@ require (
 	github.com/rancher/dynamiclistener v0.6.0-rc2
 	github.com/rancher/kubernetes-provider-detector v0.1.5
 	github.com/rancher/lasso v0.0.0-20240923125127-ae858d002589
-	github.com/rancher/norman v0.0.0-20240708202514-a0127673d1b9
+	github.com/rancher/norman v0.0.0-20240822182819-60ccfabc4ac5
```

Comment:
This points to a commit of norman on the main branch, not the `release/v2.9` branch. Any reason why this is needed?